src/koalakeys/templates/cheatsheets/assets/cheatsheets.css

```
@@ -0,0 +1,468 @@
+	padding-left: 200px;
+}
+h1 {
+	color: var(--kk-accent);
+	text-align: center;
+	margin-bottom: 10px;
+	font-weight: 700;
+	font-size: 2.2rem;
+	position: absolute;
+	left: 50%;
+	right: 0;
+	margin: 0;
```

Stylelint (17.13.0)

[error] Overridden property "margin-bottom" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)